host-bmc/utils.cpp

```diff
@@ -77,6 +81,17 @@ void addObjectPathEntityAssociations(
     }
 
     std::string entityName = entityMaps.at(node_entity.entity_type);
+    std::string entityInstanceNum =
+        std::to_string(node_entity.entity_instance_num);
+
+    /* If we get entity type as chassis, it means we encountered a mex or
+     * splitter chassis*/
```

Comment:
terms like mex and splitter are IBM specific , avoid quoting those in comments since community does not understand them.